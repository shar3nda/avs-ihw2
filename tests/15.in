;Y;ASxTR@~./w5*rCFLE\E$m-f0ht2%z3'i`j}I_ZW^7Z0mwX~&Oxp;d,@_egC:$~8cYo,C

eH] />