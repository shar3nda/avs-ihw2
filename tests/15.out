Lowercase: 754, uppercase: 827